buildSrc/src/main/java/org/opensearch/gradle/BwcVersions.java

```diff
@@ -268,8 +268,9 @@ private String getBranchFor(Version version) {
         // The current version is being worked, is always unreleased
         unreleased.add(currentVersion);
 
-        // version 1 is the first release, there is no previous "unreleased version":
-        if (currentVersion.getMajor() != 1) {
+        if (currentVersion.getMajor() == 1 && currentVersion.getMinor() == 0) {
+            // there are no previous unreleased versions before 1.0.0
```

Comment:
I think we just need to add a condition before it and leave the rest as it is.  The check can be something like 
```
// No unreleased versions for 1.0.0
if (currentVersion.equals(Version.fromString("1.0.0"))) {
    return unmodifiableList(unreleased);
}
```